skll/config.py

```diff
@@ -582,6 +581,23 @@ def _parse_config_file(config_path, log_level=logging.INFO):
     # using the defaults?
     do_grid_search = config.getboolean("Tuning", "grid_search")
 
+    # Check if `param_grids` is specified, but `grid_search` is False
+    if param_grid_list and not do_grid_search:
+        logger.warning('"param_grids" was specified despite the fact that '
+                       '"grid_search" was not specified or specified as '
+                       'False.')
+
+    # Warn user about potential conflicts between parameter values
+    # specified in `fixed_parameter_list` and values specified in
+    # `param_grid_list` (or values passed in by default) if
+    # `do_grid_search` is True
+    if do_grid_search and fixed_parameter_list:
+        logger.warning('"grid_search" is set to True and "fixed_parameters"'
```

Comment:
Minor rephrasing: `Note that "grid_search" is set to True and "fixed_parameters" is also specified. If there is a conflict between the grid search parameter space and the fixed parameter values, the fixed parameter values will take precedence.`